cortx-motr.spec.in

```diff
@@ -91,6 +91,7 @@ BuildRequires:  perl-autodie
 BuildRequires:  systemd-devel
 BuildRequires:  python-devel
 BuildRequires:  libedit-devel
+BuildRequires:  libfabric-devel
```

Comment:
I think we already discussed this: motr should not *require* libfabric, but it should use it when it is available. This is because opensource version should work without libfabric (with sock).